include/controller/c/webots/supervisor.h

```diff
@@ -125,6 +127,9 @@ void wb_supervisor_node_add_force(WbNodeRef node, const double force[3], bool re
 void wb_supervisor_node_add_force_with_offset(WbNodeRef node, const double force[3], const double offset[3], bool relative);
 void wb_supervisor_node_add_torque(WbNodeRef node, const double torque[3], bool relative);
 
+int wb_supervisor_node_get_field_count(WbNodeRef node);
```

Comment:
I think it would be better to follow the style used by the Robot API for the devices:
https://www.cyberbotics.com/doc/reference/robot#wb_robot_get_device_by_index

and rename these functions
```suggestion
int wb_supervisor_node_get_number_of_fields(WbNodeRef node);
```
same for `wb_supervisor_node_get_proto_field_count`.